src/test/e2e-emulated/contracts/stdlib.tact

```diff
@@ -58,4 +58,70 @@ contract StdlibTest {
     get fun parseVarAddress(slice: Slice): VarAddress {
         return parseVarAddress(slice);
     }
-}
\ No newline at end of file
+
+    get fun builderDepth(bl: Builder): Int {
+        return bl.depth();
+    }
+
+    get fun skipLastBits(sc: Slice, n: Int): Slice {
+        return sc.skipLastBits(n);
+    }
+
+    get fun firstBits(sc: Slice, n: Int): Slice {
+        return sc.firstBits(n);
+    }
+
+    get fun lastBits(sc: Slice, n: Int): Slice {
+        return sc.lastBits(n);
+    }
+
+    get fun sliceDepth(sc: Slice): Int {
+        return sc.depth();
+    }
+
+    get fun addressNone(): Address? {
```

Comment:
Why does it return "Maybe Address" and not "Address" ?

Reply:
Because the way Tact currently work with addresses is that "null" as an address is actually addressNone

Reply:
Well, I checked the sources and you are 100% right, but maybe we should change it?
Or change the tlb schemes generation, because 
```
struct Test {
    my_addr: Address?;
}

```

results
```
## Test
TLB: `_ my_addr:Maybe address = Test`
Signature: `Test{my_addr:Maybe address}`
```
In .md file (and maybe in some other places)

Reply:
it's an old problem in Tact, which was blocking multiple issues in the past. I remember some old comments in address-related issues about that.

My opinion is that we should rework addresses, because currently the way they work is not intuitive and doesn't fit into how other types work.
Basically, `addr_none` should be a valid part of `Address` type and should be checked by devs against `addressNone()` (and not `null`). And `Address?` should mean `Maybe Address`, which also could contain `addr_none`.
@anton-trunov what do you think?

Reply:
> addr_none should be a valid part of Address type and should be checked by devs against addressNone() (and not null).

yeah, this is how it should work, indeed

> And Address? should mean Maybe Address

also agreed

> which also could contain addr_none.

this one I'm not sure about, I'd say addr_none in the optional version should be `000`, what do you think?

Reply:
> this one I'm not sure about, I'd say addr_none in the optional version should be `000`, what do you think?

well yes, that's what I meant. I was talking about the possibility of the case when `Address?` is equal to `addressNone()`.

Reply:
@Gusarich Maybe you could take care of this change first (allowing `addr_none` as an inhabitant of the `Address` type) and then we merge this PR with the corresponding changes